Source/BugsnagCrashReport.m

```diff
@@ -630,9 +630,12 @@ + (instancetype)errorDataFromThreads:(NSArray *)threads {
             }
             NSString *contentValue = data[@"value"];
 
-            if (!contentValue) {
+#pragma clang diagnostic push
+#pragma ide diagnostic ignored "OCDFAInspection"
```

Comment:
suppresses warnings on a redundant nsstring type check, which actually isn't redundant when `contentValue` is `NSNull`